stream_chat/tests/test_client.py

```diff
@@ -220,6 +224,8 @@ def test_query_flag_reports(
         channel.send_message(msg, random_user["id"])
         client.flag_message(msg["id"], user_id=server_user["id"])
 
+        wait_for(lambda: client._query_flag_reports(message_id=msg["id"]), timeout=10)
```

Comment:
The callable parameter here is called `condition`, but what is the condition? I don't understand what it is waiting for here.